sig-device-iot/README.md

```diff
@@ -6,8 +6,9 @@ The [charter](./charter.md) defines the scope and governance of the Device IoT S
 
 ## Meetings
 
-* Regular SIG Meeting: [Friday at 10:00 UTC+8](https://zoom.us/j/4167237304) (biweekly, starts on Sep. 25th 2020). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
- * [Meeting notes and Agenda](https://docs.google.com/document/d/1PeuZzfLjU8oIduKI2nFeckKAhhckME4tqCZ33WmORBI/edit#heading=h.252sw1n3b1vb).
+* Regular SIG Meeting: [Friday at 15:00 UTC+8](https://zoom.us/j/4167237304) (biweekly, starts on Sep. 25th 2020). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
```

Comment:
The thetimezoneconverter need to be updated.

Reply:
done